### DISCUSSION TITLE
WSL 2 Ubuntu  20.04.6 LTS unable to resolve dns name

### DISCUSSION THREAD

**Discussion by fco-code-org** [microsoft/WSL#11525] [General]
Hello,

nslookup www.google.com
I got: ;; connection timed out; no servers could be reached

I have read a lot of thing about this , /etc/wsl.conf in order to keep the nameserver settings, delete the symbolic link and recreate the /etc/resolv.conf with nameserver entries and make it permanent with chattr +i apply on it.

None of what I read over internat has worked for me :-( 

More this problem seems quite random. On one PC it works and on another one it doesn't work even with the same wsl and Ubuntu version.
Could it be due to an interaction between WLS/Ubuntu network stack and Windows 10 network stack ?

 One of the try I made which didn't work was:
-  remove the symbolic link /etc/resolv.conf
- create an /etc/resolv.conf with 2 nameserver entries 8.8.8.8 and 1.1.1.1
- chattr +i /etc/resolv.conf
- create an /etc/wsl.conf file with the following content
[network]
generateResolvConf=false
- wsl --shutdown
- open a brand new Ubuntu session
- run nslookup www.google.com: same story :-( 

Is there somewhere a step by step solution description to fix this issue ? 

Best Regards.

**Comment by fco-code-org** [comment]
Hello,

Any reply ? solution?

Best Regards.

**Comment by kardeiz** [comment]
So, there might be a new problem with WSL networking: https://github.com/microsoft/WSL/issues/11512. 

The steps you describe used to work for me until 5-10 days ago.

Are you on a company VPN or using Cisco AnyConnect or similar?

**Reply by fco-code-org** [reply]
Hello,

Yes I am on a company VPN and proxy.
When it worked on other comuters (eg personal ones), I am on Internet (eg no VPN usage at all).

Best regards.
